test/Serilog.Tests/Parsing/MessageTemplateParserTests.cs

```diff
@@ -20,12 +21,38 @@ static void AssertParsedAs(string message, params MessageTemplateToken[] message
                 messageTemplateTokens);
         }
 
+        [Fact]
+        public void ANullMessageIsAException()
+        {
+            Assert.Throws<ArgumentNullException>(() => Parse(null));
+        }
+
         [Fact]
         public void AnEmptyMessageIsASingleTextToken()
         {
             var t = Parse("");
             Assert.Single(t);
```

Comment:
These assertions return values, so this can be written as:

```csharp
            var single = Assert.Single(t);
            var tt = Assert.IsType<TextToken>(single);
```

or even (questionably 😀): 

```csharp
            var tt = Assert.IsType<TextToken>(Assert.Single(t));
```

Reply:
I didn't know that.